doc/user_guide/transformations.rst

```diff
@@ -396,6 +396,46 @@ Let's plot the results side by side:
     )
     plt.show()
 
+
+Total gradient amplitude (aka Analytic Signal)
+----------------------------------------------
+
+We can also calculate the total gradient amplitude of any magnetic anomaly grid.
+This transformation consists in obtaining the amplitude of the gradient of the
+magnetic field in all the three spatial directions by applying  
+
+.. math::
+    |A(x, y)| = \sqrt{\left( \frac{\partial M}{\partial x} \right)^2 + \left( \frac{\partial M}{\partial y} \right)^2 + \left( \frac{\partial M}{\partial z} \right)^2}
```

Comment:
One other thing I noticed was the difference in the definition of tga between the tutorial and the docstring of the function. Here the absolute value is used, while in the docstring it only appears as $A(x, y)$. I think we should use the same definition in both places. Since the square root is conventionally returning the positive values, I think we can spare the absolute value bars here. What do yo think?

Reply:
I agree. Probably was a bad copy/paste.

Reply:
And now I solved it (I think)